examples/tutorial/x/blog/handlers_test.go

```diff
@@ -1,14 +1,166 @@
 package blog
 
 import (
+	"context"
+	"encoding/hex"
+	"fmt"
+	"os"
 	"testing"
+
+	"github.com/iov-one/weave"
+	"github.com/iov-one/weave/store"
+	"github.com/iov-one/weave/x"
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	weaveCtx weave.Context
+	txs      map[string]weave.Tx
+	handlers map[string]weave.Handler
+	objects  map[string]weave.Persistent
 )
 
+func toWeaveAddress(addr string) weave.Address {
+	d, err := hex.DecodeString(addr)
+	if err != nil {
+		panic(err)
+	}
+
+	return d
+}
+
+func newContext(helpers x.TestHelpers) weave.Context {
+	ctx := context.Background()
+	ctx = weave.WithHeight(ctx, 100)
+	return ctx
+}
+
+func newAuth(helpers x.TestHelpers, ctx weave.Context) x.Authenticator {
```

Comment:
There is no way to pass in or get out the address or public key that is tied to this auth?

I mean, it would be good to be able to set up the tests with control such that authorized user (from context) is equal to the author of the blog, or different. Here the authorized user is random, so guaranteed not to match.